tests/chart_tests/test_stan_sts.py

```diff
@@ -37,9 +37,10 @@ def test_stan_statefulset_defaults(self, kube_version):
         }
 
         assert all(c["securityContext"] == {"runAsNonRoot": True} for c in c_by_name.values())
-        assert doc["spec"]["template"]["spec"]["nodeSelector"] == {}
-        assert doc["spec"]["template"]["spec"]["affinity"] == {}
-        assert doc["spec"]["template"]["spec"]["tolerations"] == []
+        spec = doc["spec"]["template"]["spec"]
+        assert spec["nodeSelector"] == {}
+        assert spec["affinity"] == {}
+        assert spec["tolerations"] == []
```

Comment:
We should replace the values on line 96 in this file with global_platform_node_pool_config